docs/db_dev_process/minitests/index

```diff
@@ -0,0 +1 @@
+../../../minitests/
```

Comment:
What is this?

Reply:
It is a "fake" symbolic link that is necessary in this particular case (as well as for the fuzzer sections) where the README.md file is absent from the [minitest](https://github.com/SymbiFlow/prjxray/tree/master/minitests) directory.
Without the sym-link the recommonmark package will treat [this link](https://github.com/SymbiFlow/prjxray/#minitests) as an absolute path (as it does not appear in the xref-links, nor in the known url schemas).

It can be removed as soon as a README.md file is created in both the directories.